client/components/Dashboard/RecentScenarios.jsx

```diff
@@ -0,0 +1,69 @@
+import './Dashboard.css';
+
+import { Button, Icon } from '@components/UI';
+import { Container, Divider, Header, List, Segment } from '@components/UI';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import ScenarioCard from '@components/ScenariosList/ScenarioCard';
+import { getRecentScenarios } from '@actions/scenario';
+
+const ScenarioContent = ({ scenarios = [] }) => {
+  if (scenarios.length) {
+    return (
+      <List className="dashboard-grid">
+        {scenarios.map((scenario, index) => {
+          return (
+            <List.Item key={`scenario-card-${scenario.id}-${index}`}>
+              <ScenarioCard scenario={scenario} />
```

Comment:
Needs an `onClick` handler that will set (some boolean state bit for) `open=true|false` and the `selected` scenario to: 

```js
<ScenarioDetailModal
  open={open}
  onClose={onScenarioModalClose}
  scenario={selected}
/>
```

`onScenarioModalClose` should just set `open` to `false` and clear the `selected` scenario.